tests/charm++/jacobi3d/jacobi3d.C

```diff
@@ -99,14 +103,15 @@ class Main : public CBase_Main {
 	std::vector<std::pair<double,int> > times;
 
     Main(CkArgMsg* m) {
-      if ( (m->argc != 3) && (m->argc != 7) && (m->argc != 5) && (m->argc != 9) ) {
+      if ( (m->argc != 3) && (m->argc != 7) && (m->argc != 5) && (m->argc != 9) && (m->argc != 10) ) {
```

Comment:
Shouldn't the addition of `msg_overhead` add an `m->argc != 8` case, too (when `maxIter` and `ckptFreq` are not specified)?

Reply:
Yes, but if one were to provide the parameters as printed in the message on line 108, `argc` would be 8, so it seems like that should be a supported case.

Reply:
My bad. Misunderstood your comment. Ideally this case was not considered, since I did not take a case where we could pass overhead without passing maxiter and checkpoint frequency as parameter. See the if else lines that follows. But will fix this in the next commit.